plugins/catalog-backend/src/processing/DefaultCatalogProcessingEngine.ts

```diff
@@ -116,11 +116,16 @@ export class DefaultCatalogProcessingEngine implements CatalogProcessingEngine {
             });
           }
 
+          const location =
+            unprocessedEntity?.metadata?.annotations?.[
+              'backstage.io/managed-by-location'
```

Comment:
maybe use the `ANNOTATION_LOCATION` constant

Reply:
Also, does the locationKey make a lot of sense here instead of the annotation? For regular url ingested things i think it's going to be the same value right? Also safer to get, because at this point in the code it could be that the entity failed validation fatally and like `metadata.annotations` could be the digit `7` rather than an object.

Reply:
I think entity providers will not use the location url as locationKey 🤔

Reply:
Yep but I thought it might actually be a strength to have that represented too. But it's fine!

Reply:
ah, I see, additionally and not alternatively. I could file another PR quickly to add it, too